brage-import/src/main/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumer.java

```diff
@@ -258,9 +260,23 @@ private UriWrapper discardedFilesReportUri(Publication publication, S3Event s3Ev
     }
 
     private BrageMergingReport persistInDatabaseAndCreateMergeReport(Publication publicationForUpdate,
-                                                                     Publication existinPublication) {
-        var newImage = resourceService.updatePublication(publicationForUpdate);
-        return new BrageMergingReport(existinPublication, newImage);
+                                                                     Publication existingPublication) {
+        var resourceOwner = publicationForUpdate.getResourceOwner();
+        var publisher = publicationForUpdate.getPublisher();
+        if (customerIsNotOverridden(existingPublication, resourceOwner, publisher)) {
+            var newImage = resourceService.updatePublication(publicationForUpdate);
+            return new BrageMergingReport(existingPublication, newImage);
+        } else {
+            resourceService.permanentlyRemovePublication(existingPublication);
+            resourceService.createEmptyPublicationForCustomer(resourceOwner, publisher, existingPublication.getIdentifier());
+            var newImage = resourceService.updatePublication(publicationForUpdate);
+            return new BrageMergingReport(existingPublication, newImage);
```

Comment:
This can be combined to one method in resourceService, but I think it is best to keep it like this, to make it visible whats going on. 
Decided to use existing method when "updating" publication.